src/components/AppMap.ts

```diff
@@ -834,9 +836,50 @@ export default class AppMap extends mixins(MixinUtil) {
       marker.data.getMarker().addTo(this.map.m);
     }
 
+    this.updateTooltip();
     this.searching = false;
   }
 
+  yTooltipOnEach(marker: any) {
+    let m: any = marker.getMarker();
+    if (!('_tooltip' in marker.obj)) {
+      // @ts-ignore
+      let tt = m.getTooltip();
+      marker.obj._tooltip = tt.getContent();
+      marker.obj._tooltip_options = tt.options;
+    }
+    m.unbindTooltip();
+    m.bindTooltip(`${marker.obj.pos[1]}`, { permanent: true });
+    m.openTooltip();
+  }
+
+  yTooltipOffEach(marker: any) {
```

Comment:
disableYTooltip? Funnily enough this reminds me of Nintendo's naming conventions for enabling and disabling things (they use on/off as a verb) :upside_down_face: